tests/integration/test_provider.py

```diff
@@ -244,6 +244,9 @@ async def test_connection_updated_on_tls_enabled(ops_test: OpsTest, app_charm: P
         model_full_name=ops_test.model_full_name,
         endpoint="kafka-client-consumer",
     )
+
     assert provider_data["tls"] == "enabled"
     assert "9093" in provider_data["endpoints"]
     assert "2182" in provider_data["zookeeper-uris"]
+    assert "test-prefix" in provider_data["consumer-group-prefix"]
+    assert "test-topic" in provider_data["topic"]
```

Comment:
I think you could add a check in L69-L74 checking for the expected CONSUMER_GROUP_PREFIX in the ACLs returned, just to confirm Kafka actually set the right ones.

Reply:
Done, thanks for the suggestion!